src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -557,6 +558,11 @@ inlineToMarkdown opts lnk@(Link attr@(ident,classes,kvs) txt (src, tit)) = do
       | useAuto -> return $ literal srcSuffix
       | otherwise -> return linktext
     _ | useAuto -> return $ "<" <> literal srcSuffix <> ">"
+      | useAuto && useWikilink -> return $ "[[" <> literal srcSuffix <> "]]"
+      | useWikilink && isEnabled Ext_wikilinks_title_after_pipe opts -> return $
+        "[[" <> literal src <> "|" <> linktext <> "]]"
```

Comment:
I think there might be a problem using `linktext` here.
In the commonmark extension, we parse the page title as raw text, so e.g. `*hi*` is not emphasized text.
It's not entirely clear how we should handle the case where we have a Link designated as a "wikilink" but the link text contains formatting.  One option would be to use `stringify`.  Or perhaps we could do what you're doing here, which would create raw markdown formatting.

Reply:
I changed this to use stringify.